src/test/java/edu/kis/powp/jobs2d/VisitorTest.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d;
+
+import edu.kis.powp.jobs2d.command.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Iterator;
+import java.util.logging.Logger;
+
+public class VisitorTest implements ActionListener {
+	private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+	@Override
+	public void actionPerformed(ActionEvent e) {
+		CompoundCommand compoundCommand = new CompoundCommand("Rectangle");
+		compoundCommand.addCommand(new SetPositionCommand(-50, -50));
+		compoundCommand.addCommand(new OperateToCommand(-50, 50));
+		compoundCommand.addCommand(new OperateToCommand(50, 50));
+		compoundCommand.addCommand(new OperateToCommand(50, -50));
+		compoundCommand.addCommand(new OperateToCommand(-50, -50));
```

Comment:
Extract this code and provide a simple factory to generate this command, then add "user test" option to load it to the CommandManager (not to test the visitor).